### PR TITLE
AC Automated Test - LRAC-10590 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10590

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path [hunk 1]
```diff
@@ -35,6 +35,11 @@
 	<td>//span[contains(@class,'loading-animation')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TABLE_COLUMN_NUMBERS</td>
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path:39]: hi @ViniLopes20 , I think we could improve this path name to something more precise, like `TABLE_CELL_VALUE`? What do you think?

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro [hunk 2]
```diff
@@ -914,4 +914,27 @@ definition {
 		WaitForElementNotPresent(locator1 = "ACUtils#LOADING_ANIMATION");
 	}
 
+	macro viewTableColumnNumbers {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro:913]: hi @ViniLopes20 , I would like to suggest the same improvement I did to the path name to this macro name, something like `viewTableCellValue` or `viewTableValues`. What do you think?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro:913]: Hey @alexcandidos, What do you think if the macro name was `viewTableInformation`?

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path:39]: Hey @alexcandidos, What do you think if the path name was `TABLE_CELL_INFORMATION`?

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path:39]: hey @ViniLopes20 , sounds good to me!

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
Hey @alexcandidos, thanks for the suggestions, I made the adjustments

**Comment by ViniLopes20** [comment]
It's covered by https://github.com/liferay-qa/liferay-portal/pull/1602

**Pull request closed without merging**